app/presenters/two_factor_options_presenter.rb

```diff
@@ -26,6 +26,15 @@ def options
       backup_code_option + webauthn_option + piv_cac_option
   end
 
+  def selected_options
```

Comment:
It took me a moment to realize how and why this is different from `#options`. I think "selected" kinda helps, although at first I thought that had more to do with checked options, or the distinction between new account setup vs. sign-in.

Not sure if some variation of "all options" or "unfiltered options" might work, or a YARD-style comment block explaining how what the method is doing that's distinct from `#options` ?

Reply:
I went with both a comment and clarified method label.